src/server/utils.ts

```diff
@@ -4,10 +4,10 @@ import * as jwt from "jsonwebtoken";
 
 const SALT = "TL]{~eeo=u8J>j>@th8Psh4FQZ:^Wz)UMi;/vXst";
 
-export function handleError(err: Error, res: express.Response, statusCode: number = 400) {
+export function handleError(err: Error, res: express.Response, statusCode: number = 200) {
     console.log(err.message);
```

Comment:
do we really want to print each error? without any metainformation? (uses, accessed resource, ...)